starters/cms/src/data.ts

```diff
@@ -141,14 +137,19 @@ export async function syncCollection(
         })
     }
 
-    await collection.setFields(sanitizedFields)
     await collection.removeItems(Array.from(unsyncedItems))
     await collection.addItems(items)
 
     await collection.setPluginData(PLUGIN_KEYS.DATA_SOURCE_ID, dataSource.id)
     await collection.setPluginData(PLUGIN_KEYS.SLUG_FIELD_ID, slugField.id)
 }
 
+export const syncMethods = [
+    "ManagedCollection.removeItems",
+    "ManagedCollection.addItems",
+    "ManagedCollection.setPluginData",
+] as const satisfies ProtectedMethod[]
```

Comment:
Is the term _Protected_ often associated with requiring permissions? It's new to me. We could also go for something more descriptive? e.g. `MethodRequiringPermission`. Wdyt?

Reply:
Btw. Thanks for moving the type to here. Much better ❤️

Reply:
Resolved is Slack, keeping as-is 😌